### PR TITLE
Updating simple-over-search-bar video snippet so video fits in alloca…

### PR DESCRIPTION
**Pull request by ScottDowne**
…ted space and play button icon is centered.

### CODE REVIEW

**Comment by ScottDowne** [comment]
@glogiotatidis Hey, me again :)

I updated the generic video snippet so the video fits in this space: http://people.mozilla.org/~shorlander/snippet-best-practices/about-home-layout-flexible-space.png
about-home-layout-flexible-space

I also centered the play button icon.

Thoughts?

**Comment by glogiotatidis** [comment]
scanned it quickly, looks good. will review and merge tomorrow. thnx!

File: generic-templates/video-snippets/simple-over-search-bar.html [hunk 1]
```diff
@@ -41,21 +40,19 @@
...
-    height: 310px;
+    height: 192px;
 }
 
 #logo-replacement-{{ snippet_id }} .video-container {
-    width: 480px;
-    position: absolute;
-    left: 84px;
-    top: 15px;
+    width: 100%;
```
Comment (glogiotatidis) [inline generic-templates/video-snippets/simple-over-search-bar.html]: setting width to 100% displays a gray box until video loads.

**Comment by glogiotatidis** [comment]
Search bar slightly moves when the snippet loads.

**Comment by glogiotatidis** [comment]
Please always add a comment listing the variables, their type and their description

**Comment by glogiotatidis** [comment]
Also keep git commit messages small, ideally less than 80 chars. It's a very good practice. You can always write detailed descriptions bellow the commit message in git.

**Comment by glogiotatidis** [comment]
Fixed the comments and merged https://github.com/mozilla/snippets/commit/9ab7688f20b3a7be6166f3fd201372022d32c280

Thanks Scott

**Comment by jkt230** [comment]
Hi @glogiotatidis @ScottDowne ,
I'm using the "Simple over Search Bar" video template and the size of the video is still 480 x 270. I think the size was supposed to be adjusted to 341 x 192 or to fit the dimensions that Stephen has set out here: http://people.mozilla.org/~shorlander/snippet-best-practices/about-home-layout-flexible-space.png.

Also the play button looks the same. So maybe I'm using the wrong template?

Giorgos if you want to take a look at Snippet 5642 that would be great!

screen shot 2016-06-28 at 10 32 24 am

**Comment by ScottDowne** [comment]
The screenshot there looks to also not have the fix to the position of the play button. The icon is not centered, and that was also fixed. So it looks like it's still the old template before the patch.

**Comment by ScottDowne** [comment]
But simple-over-search-bar is the correct template.

**Comment by glogiotatidis** [comment]
My bad: I copied the template into "Share video over search bar" instead of "Simple video..." anyway I restored the "Share" snippet and updated the "Simple" one.

Looks ok now.

**Comment by jkt230** [comment]
It works, thanks!

**Pull request closed without merging**